src/plugins/tools/apipTextToSpeech/plugin.js

```diff
@@ -316,6 +318,10 @@ export default pluginFactory({
 
                 getTTSComponent().setMediaContentData(ttsApipData);
                 this.show();
+                if (!this.getState('sleep')) {
+                    this.setState('enabled', true);
+                    toggleTool();
```

Comment:
This function is calling `enablePlugin()`, which is recreating the keyNavigation. This leads to inconsistency and make the unit tests failing.

Reply:
line added to test, test is passed